src/Mod/TechDraw/TechDrawTools/CommandAxoLengthDimension.py

```diff
@@ -108,6 +113,30 @@ def Activated(self):
                 distanceDim.Arbitrary = True
                 distanceDim.Label = distanceDim.Label.replace('Dimension','Dimension3D')
                 distanceDim.FormatSpec = fomatted3DValue
+            if (
```

Comment:
Lots of curious magic numbers here.   Some of them look like +/- 30 degrees.  Maybe a comment?